tests/pipeline/test_engine.py

```diff
@@ -1562,3 +1567,51 @@ def test_no_groupby_maximum(self):
                         .reset_index(level=1, drop=True))
 
         assert_equal(groupby_max, pipeline_max)
+
+
+class ResolveDomainTestCase(zf.ZiplineTestCase):
+
+    def test_resolve_domain(self):
+        engine_generic = SimplePipelineEngine(
+            None, None, default_domain=GENERIC
```

Comment:
I'd probably use `NamedExplodingObject` for these so that we raise a clear error if something in the engine starts depending on the asset finder or countries in loader dispatcher at construction time.